_posts/2025-03-18-GPU-Accelerated-Vector-Search-OpenSearch-New-Frontier.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "GPU-accelerated vector search in Opensearch: A new frontier" 
+authors:
+  - navneev
+  - cnolet
+  - kshitizgupta21
+  - dylantong
+  - nwstephens
+  - vamshin
+date: 2025-03-03
```

Comment:
@navneet1v - Please update the publish date to what we discussed via Slack.

Reply:
Sorry my bad.. will update it

Reply:
done